includes/wf_crm_admin_form.inc

```diff
@@ -1757,36 +1759,45 @@ class wf_crm_admin_form {
       $handler_configuration['settings'] = $this->settings;
       $handler->setConfiguration($handler_configuration);
 
-      $webform_elements = $this->webform->getElementsDecoded();
+      $webform_element_manager = \Drupal::getContainer()->get('plugin.manager.webform.element');
       foreach ($enabled as $enabled_key => $enabled_element) {
-        // Add new items to the webform.
-        if (isset($webform_elements[$enabled_key])) {
+        if (!is_array($enabled_element)) {
+          // If this is a string, it is not a new element. However, this
+          // probably needs to be revisited.
           continue;
         }
-        $webform_elements[$enabled_key] = [
-          '#title' => (string) $enabled_element['name'],
+        // Webform uses YAML dump, which dies on FormattableMarkup.
+        $enabled_element['extra'] = array_map(
+          [$this, 'stringifyFormattableMarkup'],
+          empty($enabled_element['extra']) ? [] : $enabled_element['extra']
+        );
+        $element_plugin = $webform_element_manager->getElementInstance([
           '#type' => $enabled_element['type'],
-          '#webform_civicrm' => $enabled_key,
-          // Webform uses YAML dump, which dies on FormattableMarkup.
-          '#extra' => array_map([get_called_class(), 'stringifyFormattableMarkup'], empty($enabled_element['extra']) ? [] : $enabled_element['extra']),
-          '#weight' => empty($enabled_element['weight']) ? 0 : $enabled_element['weight'],
-        ];
+        ]);
+
+        $stub_form = [];
+        $stub_form_state = new FormState();
```

Comment:
I don't think it matters, here, because we just need a FormStateInterface object and values assigned. We we did a SubFormState I'd have to mock the form element parent arrays.

Reply:
Ok! Just wanted to make sure we weren't missing something important, since SubFormState does a tiny bit of magic to really make the illusion of a subform work properly. I'll mark this one as resolved.